src/seedu/addressbook/commands/Command.java

```diff
@@ -15,7 +15,8 @@
     protected AddressBook addressBook;
     protected List<? extends ReadOnlyPerson> relevantPersons;
     private int targetIndex = -1;
-
+    protected final boolean COMMAND_IS_MUTATING = true;
+    protected final boolean COMMAND_NOT_MUTATING = false;
```

Comment:
Though we usually emphasize using constants instead of magic values. This is abit too fine grained as the method's name itself is self explanatory.